python/docs/source/guide/data_types.md

```diff
@@ -73,6 +73,16 @@ For example, `{name: string, age: u32 }` is a record type with two fields and `{
 
 NOTE: Record types may be nested.
 
+## Collection Types
+
+Kaskada also supports collections -- lists and maps.
```

Comment:
Well, in some cases. https://github.com/kaskada-ai/kaskada/issues/494

Reply:
That's OK for the purposes of the docs.